scripts/exgdas_global_marine_analysis_run.sh

```diff
@@ -108,6 +131,12 @@ concatenate_bump 'bump3d'
 clean_yaml var.yaml
 $APRUN_SOCAANAL $JEDI_BIN/soca_var.x var.yaml 2>var.err
 
+
+# increments update for MOM6
+( socaincr2mom6 `ls -t ${DATA}/Data/ocn.*3dvar*.incr* | head -1` ${DATA}/INPUT/MOM.res.nc ${DATA}/soca_gridspec.nc ${DATA}/Data/inc.nc )
```

Comment:
ok, I see what the previous comment was for ... Maybe that's fine for now, but the increment name should be constructed from cdate-da window instead of `ls -t` ... I write this and I've been guilty of cutting corners the same way throughout this repo!